docs/src/modules/components/ApiPage/list/ExpandableApiItem.tsx

```diff
@@ -188,7 +188,7 @@ export default function ExpandableApiItem(props: ExpandableApiItemProps) {
       ownerState={{ type }}
       {...other}
       className={clsx(
-        `MuiApi-item-root${isExtendable ? ' MuiApi-item-header-extendable' : ''}`,
+        `MuiApi-item-root${isExtendable ? 'MuiApi-item-header-extendable' : ''}`,
```

Comment:
Looks like this change is wrong as it removes the space between class names.

Reply:
Nice finding. Agree, it looks like a bug in the plugin itself.

Reply:
Issue created https://github.com/tailwindlabs/prettier-plugin-tailwindcss/issues/337.